models/articles.model.js

```diff
@@ -32,6 +32,19 @@ exports.selectArticleById = (id) => {
 		});
 };
 
+exports.selectCommentsForArticleId = (id) => {
+	return db
+		.query(`SELECT comment_id, votes, created_at, author, body FROM comments WHERE article_id = $1;`, [
+			id
+		])
+		.then(({ rows }) => {
+			if (rows.length === 0 || rows === undefined) {
+				return Promise.reject({ status: 404, msg: 'Article not found' });
```

Comment:
if the rows.length === 0, it's possible that article does exist but no comments related to it